scripts/tag-versioned-docker-images.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# This script is used to tag docker images with a specific commit or tag the
+# docker images as ":stable" with the --stable flag.
+
+# List of docker tags to update with git versioned tag.
+DOCKER_IMAGES=(
+  # Beacon chain images
+  "gcr.io/prysmaticlabs/prysm/beacon-chain"
+  "index.docker.io/prysmaticlabs/prysm-beacon-chain"
+  # Validator images
+  "gcr.io/prysmaticlabs/prysm/validator"
+  "index.docker.io/prysmaticlabs/prysm-validator"
+  # Slasher images
+  "gcr.io/prysmaticlabs/prysm/slasher"
+  "index.docker.io/prysmaticlabs/prysm-slasher"
+)
+
+
+# Check that the current commit has an associated git tag.
+TAG=$(git describe --tags HEAD)
+TAG_COMMIT=$(git rev-list -n 1 $TAG)
+CURRENT_COMMIT=$(git rev-parse HEAD)
+
+if [[ "$TAG_COMMIT" -ne "$CURRENT_COMMIT" ]]
```

Comment:
```suggestion
if [ "$TAG_COMMIT" != "$CURRENT_COMMIT" ]
```
Otherwise it compares things as numbers (and "line 25: [[: 681e20f7a4a091df0ccacec395fd0a5912540db0: value too great for base (error token is "681e20f7a4a091df0ccacec395fd0a5912540db0")" error emitted)